lambda_src/drivers/process_smtp.py

```diff
@@ -27,6 +39,20 @@ def process_row(
     password = decrypt_if_encrypted(password)
     sender_email = sender_email or user
 
+    if auth:
+        auth_json = decrypt_if_encrypted(auth)
+        assert auth is not None
```

Comment:
when would this be the case? iirc decrypt raises an exception if it cannot decrypt, and returns `None` only if `auth` is `None`

Reply:
I tried it make it uniform with the HTTPS driver but yeah will remove the assertion in both if that's the case.